openedx/core/djangoapps/notifications/templates/notifications/digest_footer.html

```diff
@@ -37,6 +37,9 @@
                     <a href="{{notification_settings_url}}" rel="noopener noreferrer" target="_blank" style="color: black">
                         Notification Settings
                     </a>
+                    <a href="{{unsubscribe_url}}" rel="noopener noreferrer" target="_blank" style="color: black; margin-left: 1rem">
```

Comment:
Do we need any waffle flags for this feature?

Reply:
I don't think we need waffle flag for this feature. Email notifications have not been rolled out yet and `unsubscribe_url` is just a small part of it. We do have a user level waffle flag for email notifications.